src/modules/logger/params.c

```diff
@@ -66,6 +66,18 @@ PARAM_DEFINE_INT32(SDLOG_UTC_OFFSET, 0);
  */
 PARAM_DEFINE_INT32(SDLOG_MODE, 0);
 
+/**
+ * Battery-only Logging
+ *
+ * When enabled, logging will not start from boot if battery power is not detected
+ * (e.g. powered via USB on a test bench). This prevents extraneous flight logs from
+ * being created during bench testing.
+ *
```

Comment:
Add another line:
Note that this applies only to log-from-boot modes. It has no effect in arming-based modes.

Reply:
Thanks, clarified.